File: devcamp2018/app/src/main/java/com/psato/devfest/data/entity/ShowIds.java
package com.psato.devfest.data.entity;

import com.google.gson.annotations.SerializedName;

/**
 * Created by psato on 29/10/16.
 */

public class ShowIds {
    @SerializedName("trakt")
    private String mTrakt;
    @SerializedName("slug")
    private String mSlug;
    @SerializedName("tvdb")
    private String mTvdb;
    @SerializedName("imdb")
    private String mImdb;
    @SerializedName("tmdb")
    private String mTmdb;
    @SerializedName("tvrage")
    private String mTvrage;

}

File: devcamp2018/app/src/main/java/com/psato/devcamp/data/entity/ShowIds.kt
package com.psato.devcamp.data.entity

import com.google.gson.annotations.SerializedName

/**
 * Created by psato on 29/10/16.
 */

class ShowIds {
    @SerializedName("trakt")
    var trakt: String? = null
    @SerializedName("slug")
    var slug: String? = null
    @SerializedName("tvdb")
    var tvdb: String? = null
    @SerializedName("imdb")
    var imdb: String? = null
    @SerializedName("tmdb")
    var tmdb: String? = null
    @SerializedName("tvrage")
    var tvrage: String? = null

}
